src/components/facet-browser/FacetBrowserModalBody.tsx

```diff
@@ -30,6 +31,25 @@ const FacetBrowserModalBody: React.FunctionComponent<
       setOpenFacets([...openFacets, facet]);
     }
   };
+  const { track } = useStatistics();
+
+  useEffect(() => {
+    const areFiltersEmpty = Object.keys(filters).length === 0;
+    if (areFiltersEmpty) {
+      return;
+    }
+    track("click", {
+      id: 20,
+      name: "Søgning Facet",
+      trackedData: JSON.stringify(
+        mapValues(filters, (filter) => {
+          return Object.keys(filter);
+        })
+      )
```

Comment:
Consider if using `JSON.stringify` is the right way to track facets.

When we have multiple values in other situations we join array values with `, `. I wonder why we do not use the same approach here.

Reply:
It is because there are multiple facet categories that can be tracked here AND they each can have multiple values. So in order for them to make sense of it it needs to be structured data we send. Well.. or at least in this case stringified structured data.
But I could totally join the separate values into one string! Instead of an array. DIdn't think of that 🤓